backend/src/schema/resolvers/embeds/findProvider.spec.js

```diff
@@ -0,0 +1,35 @@
+import findProvider from './findProvider'
+
+describe('Vimeo', () => {
+  it('matches `https://vimeo.com/showcase/2098620/video/4082288`', () => {
+    expect(findProvider('https://vimeo.com/showcase/2098620/video/4082288')).toEqual(
+      'https://vimeo.com/api/oembed.json',
+    )
+  })
+})
+
+describe('RiffReporter', () => {
+  it('matches `https://www.riffreporter.de/flugbegleiter-koralle/`', () => {
+    expect(findProvider('https://www.riffreporter.de/flugbegleiter-koralle/')).toEqual(
+      'https://www.riffreporter.de/service/oembed',
+    )
+  })
+})
+
+describe('Youtube', () => {
+  it('matches `https://www.youtube.com/watch?v=qkdXAtO40Fo`', () => {
+    expect(findProvider('https://www.youtube.com/watch?v=qkdXAtO40Fo')).toEqual(
+      'https://www.youtube.com/oembed',
+    )
+  })
+
+  it('matches `https://youtu.be/qkdXAtO40Fo`', () => {
+    expect(findProvider(`https://youtu.be/qkdXAtO40Fo`)).toEqual('https://www.youtube.com/oembed')
+  })
+
+  it('matches `https://youtu.be/qkdXAtO40Fo?t=41`', () => {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-01T05:51:44Z" title="Thursday, August 1st 2019, 7:51:44 am +02:00">Aug 1, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,35 @@
+import findProvider from './findProvider'
+
+describe('Vimeo', () => {
+  it('matches `https://vimeo.com/showcase/2098620/video/4082288`', () => {
+    expect(findProvider('https://vimeo.com/showcase/2098620/video/4082288')).toEqual(
+      'https://vimeo.com/api/oembed.json',
+    )
+  })
+})
+
+describe('RiffReporter', () => {
+  it('matches `https://www.riffreporter.de/flugbegleiter-koralle/`', () => {
+    expect(findProvider('https://www.riffreporter.de/flugbegleiter-koralle/')).toEqual(
+      'https://www.riffreporter.de/service/oembed',
+    )
+  })
+})
+
+describe('Youtube', () => {
+  it('matches `https://www.youtube.com/watch?v=qkdXAtO40Fo`', () => {
+    expect(findProvider('https://www.youtube.com/watch?v=qkdXAtO40Fo')).toEqual(
+      'https://www.youtube.com/oembed',
+    )
+  })
+
+  it('matches `https://youtu.be/qkdXAtO40Fo`', () => {
+    expect(findProvider(`https://youtu.be/qkdXAtO40Fo`)).toEqual('https://www.youtube.com/oembed')
+  })
+
+  it('matches `https://youtu.be/qkdXAtO40Fo?t=41`', () => {
```


was this test just to give the edge case of non-alphanumeric characters in the URL?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-01T10:02:23Z" title="Thursday, August 1st 2019, 12:02:23 pm +02:00">Aug 1, 2019</time>_

---


No, I wanted to see if youtube shareable links with a time position `?t=41` cause troubles or not.